src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double sideA;
+    private double sideB;
+
+    public Rectangle(double sideA, double sideB, String color) {
+        super(color);
+        this.setSideA(sideA);
+        this.setSideB(sideB);
+    }
+
+    @Override
+    public double area() {
+        double s;
```

Comment:
No need to declare and initialize the variable 's' separately. You can directly return the result of the multiplication.